out_writeapi.go

```diff
@@ -317,15 +330,7 @@ func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int
 
 		if ((currsize + len(buf)) > config.maxChunkSize) && len(binaryData) != 0 {
 			// Appending Rows
-			stream, err := config.managedStream.AppendRows(ms_ctx, binaryData)
-			if err != nil {
-				log.Fatal("AppendRows: ", err)
-				return output.FLB_ERROR
-			}
-			*config.appendResults = append(*config.appendResults, stream)
-
-			// log.Printf("len in loop: %d", len(config.appendResults))
-			// log.Println("Done")
+			sendRequest(ms_ctx, binaryData, &config)
```

Comment:
You will have to add error-handling here, so that if the call to sendRequest fails then we return a failure code back to Fluentbit.